### DISCUSSION TITLE
Students cannot accept newly created assignments

### DISCUSSION THREAD

**Discussion by GSibinger** [community/community#135880] [GitHub Education] [answered]
### Hi everyone,

I created a new classroom and a new assignment. Went through all the steps and added a **private template repository**. After creating the assignment, a new repository was automatically created from the template, which also is **private**. Then I tried the assignment link. I got the "accept assignment" prompt, clicked it and then received this text:
> This assignment can no longer be accepted. Please contact your teacher. They might need to edit or recreate the assignment.

I did not set a deadline. The settings are the same as in our other classrooms and assignments. Only this new one cannot be accessed.

If anybody from Github is reading this, this is the classroom in question: 
https://classroom.github.com/classrooms/65162677-js_backend_2

This is the assignment: 
https://classroom.github.com/classrooms/65162677-js_backend_2/assignments/blog-crud-app

Please let me know if there's anything else you need.

**Comment by ConsoleFriend** [comment]
We're currently experiencing the exact same in one of our classrooms. Several others aren't having any issues, but at the moment everything we seem to do in the other also leads to the above mentioned error.

**Comment by Drieze** [comment]
I have the same problem when I create a new assignment without a deadline. This is the error I get:
![image](https://github.com/user-attachments/assets/bdeb1898-35e9-4446-b6db-2be2e9e734ff)

**Comment by higorifrn** [comment]
Same error in here.

This is the assignment: 
https://classroom.github.com/classrooms/164347635-ilp-2024-1/assignments/lista-de-exerc-cios-04

But, previously some students were enable to create their own repositoru from the link assignment: 
https://classroom.github.com/a/GuH9-Dd8

Now, other students are getting the same error message as mentioned by @Drieze .

**Comment by chrisflath** [comment]
Same here!

**Reply by NikoStein** [reply]
same

**Comment by acorbin3** [comment]
Yep I cam getting the same error. I re-created 2 assignments and still get the error. I think this error has happened before last May
https://github.com/orgs/community/discussions/122130

**Comment by LouisTsai-Csie** [comment]
The same error occurs here, it is weird.

I attempted to resolve this issue by editing the assignment. Following this modification, students were able to accept their assignments. However, they reported that the newly created assignments do not have the original starter code and instead generated empty repositories.

I tried to re-edit the assignment to update the starter code but was unsuccessful. Currently, all my students are unable to accept the assignment.

My starter template disappeared.

<img width="893" alt="Screenshot 2024-08-15 at 9 12 22 PM" src="https://github.com/user-attachments/assets/a5955488-bc9f-4503-b4a6-db379a33b091">

**Reply by chrisflath** [reply]
Encountered exactly the same behavior.

**Reply by LouisTsai-Csie** [reply]
Same issue back in May this year.

https://github.com/orgs/community/discussions/122130

**Comment by acorbin3** [comment, marked as answer]
Someone found a solution here

https://github.com/orgs/community/discussions/122130#discussioncomment-10351784

Basically make a change on the readme and made a commit on the generated repo for the assignment ( not the template). That will allow students to accept the assignments again!
